src/Project/countryDetails.jsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import Layout from "./Layout/Layout";
+import { Link, useParams } from "react-router-dom";
+
+
+
+const CountryDetail = () => {
+  const state = useParams();
+  const [countries, setCountries] = useState([]);
+  const [ loading, setLoading ] = useState(true);
```

Comment:
For  Boolean states, use is with state name. For example, isLoading, setIsLoading